src/components/dropdownV3/DropdownSelectorItem.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
```

Comment:
We could also use interactive [args](https://storybook.js.org/docs/writing-stories/args) instead of showing all the possible scenarios.